package-lock.json

```diff
@@ -1213,7 +1213,7 @@
         },
         "node_modules/@ts-morph/common": {
             "version": "0.12.2",
-            "resolved": "https://repo.tech.hl.uk/repository/npm/@ts-morph/common/-/common-0.12.2.tgz",
```

Comment:
xD @crpc Remember that not every developer works at HL.

Reply:
There's no chance I committed that, I'm clearly being framed here... :D